components/Anchor.js

```diff
@@ -70,6 +78,7 @@ const Link = ({ children, id, sub }) => {
 
       <Anchor href={`#${id}`}>
         <AnchorIcon />
+        <span>Anchor link:{id}</span>
```

Comment:
I'm not sure whether this is actually the best way to solve this? Wouldn't it be possible to set an `aria` attribute on the anchor itself?

It seems unclean to render a `width/height: 1px` span on each of the anchors

Reply:
I'm afraid that this is the only one way to do it

Reply:
The `title` attribute should be sufficient for all anchor tags.